sdk/python/tests/compiler/component_builder_test.py

```diff
@@ -231,19 +231,19 @@ def test_prepare_docker_tarball(self):
     test_data_dir = os.path.join(os.path.dirname(__file__), 'testdata')
     dockerfile_path = os.path.join(test_data_dir, 'component.target.dockerfile')
     Path(dockerfile_path).touch()
-    generated_tarball = os.path.join(test_data_dir, 'test_docker.tar.gz')
+    local_tarball_path = os.path.join(test_data_dir, 'test_docker.tar.gz')
 
     # check
     docker_helper = DockerfileHelper(arc_dockerfile_name='dockerfile')
-    docker_helper.prepare_docker_tarball(dockerfile_path=dockerfile_path, local_tarball_path=generated_tarball)
-    temp_tarball_handler = tarfile.open(generated_tarball)
+    docker_helper.prepare_docker_tarball(dockerfile_path=dockerfile_path, local_tarball_path=local_tarball_path)
+    temp_tarball_handler = tarfile.open(local_tarball_path)
```

Comment:
Nit: it's `handle` not `handler`. https://en.wikipedia.org/wiki/Handle_(computing) https://en.wikipedia.org/wiki/Handler#Computing